Plugins/InstallSafeDITool/InstallCLIPluginCommand.swift

```diff
@@ -111,3 +103,72 @@ struct InstallSafeDITool: CommandPlugin {
         }
     }
 }
+
+#if canImport(XcodeProjectPlugin)
+    import XcodeProjectPlugin
+
+    extension InstallSafeDITool: XcodeCommandPlugin {
+        func performCommand(
+            context: XcodeProjectPlugin.XcodePluginContext,
+            arguments _: [String]
+        ) throws {
```

Comment:
this is quite similar to the above `func performCommand(context: PackagePlugin.PluginContext, arguments _: [String]) async throws`, except our context is different and we aren't `async`.